jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -317,17 +320,20 @@ def runTest(TESTS, test_type, repo_dir){
 
 def dockerPostTest(TESTS, docker_stash_name, docker_stash_path, docker_record_stash_path, repo_dir, test_type){
     setDocker(docker_stash_name, docker_stash_path, docker_record_stash_path)
-    runTest(TESTS, test_type, repo_dir)
+    test_stack = "-stack vagrant"
+    runTest(TESTS, test_type, repo_dir, test_stack)
 }
 
 def ovaPostTest(TESTS, ova_stash_name, ova_stash_path, repo_dir, test_type){
     setOVA(ova_stash_name, ova_stash_path)
-    runTest(TESTS, test_type, repo_dir)
+    test_stack = "-stack vagrant"
```

Comment:
Wanted to leave it as vagrant for this PR (that is what it is using right now) so we can get a chance to test it in the sandbox against Post OVA and Post Docker builds.